terraform/per-project/README.md

```diff
@@ -178,6 +178,10 @@ In this section you prepare your project for deployment.
     gcloud alpha firestore databases create --region="${APP_ENGINE_LOCATION}"
```

Comment:
Similar comment as for the instruction in the distributed steps, I suggest these commands be split into to two steps.

Step 9. Create a Google App Engine app, to enable the APIs for Cloud Scheduler and Firestore.

Step 10. Create database to store the state of the Autoscaler. State can be stored in either Firestore or Cloud Spanner.